tests/cdc-wal2json/ddl.sql

```diff
@@ -12,3 +12,9 @@ alter table payment_p2022_05 replica identity full;
 alter table payment_p2022_06 replica identity full;
 alter table payment_p2022_07 replica identity full;
 commit;
+
+-- Remove the 'last_updated' trigger to prevent automatic refresh of
+-- the 'last_updated' attribute when modifying rows in the address table.
+begin;
+drop trigger last_updated on address;
```

Comment:
The trigger on `address` prevented me from executing the update statement on the `address` table because the comparison check for `000000010000000000000002.json` would fail.